lib/jsonapi/view.ex

```diff
@@ -140,12 +140,12 @@ defmodule JSONAPI.View do
         "#{url_for(data, conn)}/relationships/#{rel_type}"
       end
 
-      if Code.ensure_loaded?(Phoenix) do
+      with true <- Code.ensure_loaded?(Phoenix) do
```

Comment:
Why `with` over an `if` here?  I'm not sure we gain anything.

Reply:
@doomspork Added `raise ArgumentError` in the `else` case.  Should we keep that or remove it?  Don't mind either way, but thought, in a very rare case, might help somebody.